tools/sgen/input/FluentUIStyle.yml

```diff
@@ -288,6 +291,10 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
 AP_MSFDrawerTokens:
   backgroundClearColor: NamedColor(clear)
   backgroundDimmedColor: $Colors.Elevation.highElevation
+  drawerContentBackground: "FluentUIColor(light: $Colors.Background.neutral1, dark: $Colors.Background.neutral3)"
+  popoverContentBackground: "FluentUIColor(light: $Colors.Background.surfacePrimary, dark: $Colors.Background.surfaceQuaternary)"
+  navigationBarBackground: "FluentUIColor(light: $Colors.Background.surfacePrimary, dark: NamedColor(grey14))"
+  resizingHandleViewBackgroundColor: NamedColor(clear)
```

Comment:
Is this correct? The resizing handle should have the same background as the drawer background, no?

Reply:
They both are same colors but different tokens because they are applied to different components. I am not certain if there is a design correlation between them as I don't have design spec for UIKit components. I have mostly mapped them as 1-1 tokens.

Reply:
I think we can remove the token and use the drawer background for it. If the requirement of tokenizing it is raised, then we can add it in the future.